source/Mission.cpp

```diff
@@ -207,7 +207,11 @@ void Mission::Load(const DataNode &node)
 		else if(child.Token(0) == "assisting")
 			location = ASSISTING;
 		else if(child.Token(0) == "boarding")
+		{
 			location = BOARDING;
+			if(child.Size() > 1 && child.Token(1) == "capture override")
+				overridesCapture = true;
+		}
```

Comment:
Make this a child node of `boarding`, not an additional token in the same node.

Reply:
I wasnt sure if that was the way it should be done or not but sure will change it